src/ImageHistory.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import cockpit from 'cockpit';
+import * as utils from './util.js';
+import * as client from './client.js';
+
+import { ListingTable } from "cockpit-components-table.jsx";
+
+const _ = cockpit.gettext;
+
+const IdColumn = Id => {
+    Id = utils.truncate_id(Id);
+    // Not an id but <missing> or something else
+    if (Id.length != 12) {
```

Comment:
Not the most robust check, but I don't have any better idea either (except maybe additionally checking if it is also hexadecimal)

Reply:
An idea is to use regex, like `<[a-z]+>`

Reply:
Applied, seems to work nicely